_site/docs/tools/bf-cat.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: bf-cat
+parent: Tools
+nav_order: 1
+---
+
+bf-cat is a tool provided with buildfarm for investigating the various structures and status of your Buildfarm Cluster.
+
+Its basic usage is:
+
+`bf-cat <host[:port]> <instance-name> <hash-function> <command> [params...]`
+
+**instance-name** is the name of the specific instance to inquire about, typically configured on schedulers. A literal empty string parameter (i.e. bash: `""`) will use the default instance for a server.
+
+**hash-function** is one of MD5, SHA1, SHA256, etc selected to match the supported digest functions of an instance, and used to compute digests for content retrieved.
+
+**command** is typically one of the following, with digest parameters as <hash>/<size>, as typically represented in log entries:
+
+* **Action <digest...>**: Retrieves Action definitions from the CAS and renders them with field identifiers.
```

Comment:
Looking at the generated doc: https://luxe.github.io/buildfarm/docs/tools/bf-cat/

> ****Action** **: Retrieves Action definitions

The <digest ...> part is completely swallowed up somewhere.

Reply:
it seems that markdown will interpret them as tags which become invalid html: `<digest></digest>`.  We will need to either annotate them as code, or escape them.

Reply:
fixed